sdks/python/apache_beam/io/gcp/bigtableio_test.py

```diff
@@ -0,0 +1,218 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unittest for GCP Bigtable Split testing."""
+from __future__ import absolute_import
+from __future__ import division
+
+import logging
+import mock
+import sys
+import unittest
+
+from .bigtableio import BigtableSource
+
+try:
+  from google.cloud.bigtable import Client
+  from google.cloud.bigtable.row_data import PartialRowData
+  from google.cloud.bigtable.row_set import RowRange
+  from google.cloud.bigtable.row_set import RowSet
+  from google.cloud.bigtable.table import Table
+  from google.cloud.bigtable_v2.proto.bigtable_pb2 import SampleRowKeysResponse
+except ImportError:
+  Client = None
+  PartialRowData = None
+  RowRange = None
+  RowSet = None
+  SampleRowKeysResponse = None
+  Table = None
+
+KEYS_1 = [b'beam_key0672496', b'beam_key1582279', b'beam_key22',
+          b'beam_key2874203', b'beam_key3475534', b'beam_key4440786',
+          b'beam_key51', b'beam_key56', b'beam_key65', b'beam_key7389168',
+          b'beam_key8105103', b'beam_key9007992', b'']
+
+KEYS_2 = [b'beam_key0952711', b'beam_key2', b'beam_key2797065',
+          b'beam_key3518235', b'beam_key41', b'beam_key4730550',
+          b'beam_key54', b'beam_key6404724', b'beam_key7123742',
+          b'beam_key7683967', b'beam_key83', b'beam_key8892594',
+          b'beam_key943', b'']
+
+RANGES_DICT = {
+  '': (0, 952),
+  'beam_key0952': (952, 2000),
+  'beam_key2': (2000, 2797),
+  'beam_key2797': (2797, 3518),
+  'beam_key3518': (3518, 4100),
+  'beam_key41': (4100, 4730),
+  'beam_key4730': (4730, 5400),
+  'beam_key54': (5400, 6404),
+  'beam_key6404': (6404, 7123),
+  'beam_key7123': (7123, 7683),
+  'beam_key7683': (7683, 8300),
+  'beam_key83': (8300, 8892),
+  'beam_key8892': (8892, 9430),
+  'beam_key943': (9430, 9930),
+}
+
+SIZE_768M = 805306368
+SIZE_9984M = 10468982784
+
+@unittest.skipIf(Table is None, 'GCP Bigtable dependencies are not installed')
+@unittest.skipIf(Client is None, 'GCP Bigtable dependencies are not installed')
```

Comment:
Prob. just checking for one of these enough (may be Client) ?

Reply:
Agreed. The 'Table is None' check has been removed.